hasher-matcher-actioner/src/OpenMediaMatch/storage/postgres/impl.py

```diff
@@ -523,14 +523,35 @@ def bank_delete(self, name: str) -> None:
         database.db.session.commit()
 
     def bank_content_get(
-        self, ids: t.Iterable[int]
+        self, ids: t.Iterable[int], signal_type: t.Optional[str] = None
     ) -> t.Sequence[interface.BankContentConfig]:
-        return [
-            b.as_storage_iface_cls()
-            for b in database.db.session.query(database.BankContent)
-            .filter(database.BankContent.id.in_(ids))
-            .all()
-        ]
+        query = database.db.session.query(database.BankContent)
+
+        if signal_type is not None:
+            query = query.outerjoin(
+                database.ContentSignal,
+                database.ContentSignal.content_id == database.BankContent.id,
+            )
```

Comment:
ignorable; I think if we are going to fetch this information, doing it as part of the original query is the most efficient way to do it. I just haven't done the labor to try and prove what the access patterns are and whether it's worth optimizing for this one.